### DISCUSSION TITLE
Inquiry About Annual Hosted

### DISCUSSION THREAD

**Discussion by Michael-Merzek** [snipe/snipe-it#11628] [Q&A]
Dears 

I need to know if I choose to make an Annual Hosted with Snipe-IT from your website "https://snipeitapp.com/pricing" 
I will find the App installed or I need to install it myself?
Can I restore a backup from my free version?

Thanks

**Comment by brianhoganm** [comment]
The annual hosted and the free "open source" is exactly the same. You can restore a backup from the free version.

**Reply by brianhoganm** [reply]
Forgot to mention that I have a Server 2019 VM set up so anytime I make a major change, I first take a snapshot of the VM using Hyper-V and a backup in SnipeIT. This way, I can either roll back my VM to a good state or restore from the database. Just a nice safe measure. Hope this help.

**Reply by Michael-Merzek** [reply]
Thanks dear 
But I ask if I choose to make annual host I will start to set up the snipe it or I will receive it installed already and just upload the backup file ?

**Comment by brianhoganm** [comment]
Yes, if you provide them with the zipped file of the database backup, you
will be able to pick up from where you left off on the hosted plan.

**Reply by Michael-Merzek** [reply]
But I ask if I choose to make annual host I will start to set up the snipe it or I will receive it installed already ?

**Comment by brianhoganm** [comment]
Annual hosted means they "the company" will host the snipeit on their
servers. No need for install or server configuration. In other words it's
all set up. Once you sign up for the hosted plan, they will give a URL and
a login. I am not a hosted customer. I've only created a VM on premise.

You will have a fully functional snipeit that you can access and set up.

However, if you have a backup of your on premise snipit database, they can
help you with that as well since you are a customer paying for the hosted
plan.

**Reply by Michael-Merzek** [reply]
Thanks dear I got it thanks a lot for your useful replay  appreciate it
